### PR TITLE
to_boolean: fix conversion in case of numbers

### PR DESCRIPTION
**Pull request by D-Alex**
some of the sdf files use '1' instead of 'true' and '0' instead of
'false'

### CODE REVIEW

**Comment by doudou** [comment]
Could you add the corresponding tests ? Thanks.

**Comment by D-Alex** [comment]
Done - it also removes blanks now.

File: lib/sdf/conversions.rb [hunk 1]
```diff
@@ -46,16 +46,16 @@ def self.vector3_to_eigen(vector3)
...
-        # @param [String,#text] pose the pose as "x y z" (as expected in SDF) or
-        #   nil, in which case a zero translation is returned
+        # @param [String,#text] boolean the SDF boolean ('true','false','0',1')
         # @return [Boolean]
         def self.to_boolean(xml)
             if xml.respond_to?(:text)
                 xml = xml.text
             end
-            if xml == 'true'
+            xml.gsub!(" ","")
```
Comment (doudou) [inline lib/sdf/conversions.rb:55]: This is much better written as

```
xml = xml.strip
```

which would also not convert "fa ls     e" to "false"

**Comment by doudou** [comment]
Fixed the gsub vs. strip bug and merged.

**Pull request closed without merging**